assets/src/edit-story/components/canvas/canvasElementDropzone.js

```diff
@@ -54,7 +54,7 @@ function CanvasElementDropzone({ children }) {
           resource,
           offset: { x: offsetX, y: offsetY, w: offsetWidth, h: offsetHeight },
         } = JSON.parse(e.dataTransfer.getData('resource/media'));
-        const { x, y } = pageContainer?.getBoundingClientRect();
+        const { x, y } = fullbleedContainer?.getBoundingClientRect();
```

Comment:
I'd expect most of operations to be done against `safezoneContainer`. Why fullbleed here?